.github/workflows/welcome-first-time-contrib.yml

```diff
@@ -0,0 +1,83 @@
+#This action is centrally managed in https://github.com/asyncapi/.github/
+#Don't make changes to this file in this repo as they will be overwritten with changes made to the same file in above mentioned repo
+
+name: Welcome first time contributors
```

Comment:
This comment is a side-note from this PR. It seems this file and other workflows are all the same, with the same code. Could we move these scripts to a single file in the `community` repo, so that we use only one line in this GitHub action?
We already have that repo with all the workflows, but could we improve 🤔, any thoughts on this @magicmatatjahu ?

Reply:
yo, not sure it is worth the effort if anyway we handle replication

Reply:
ahhh they are automatically updated. Then that's even cooler, so nevermind my comment 😅 
Pretty cool stuff @derberg 🙂